students/rusty_mann/session04/mailroom_session04.py

```diff
@@ -72,15 +72,15 @@ def send_donor_email():
     donor_dict = {}
     while True:
         name = input("Please enter a donor's name in the form of 'Last name, First name' "
-            "(or 'list' to see a list of all donors, or 'menu' to exit)> ").strip()
+            "(or 'list' to see a list of all donors, or 'menu' to exit)> ").strip().lower()
         if name == "list":
             show_donor_list()
-        elif name == "menu":
+        elif name == "menu".strip().lower():
             return None
         else:
             break
     while True:
```

Comment:
Nice use of break to allow user to see the list and then enter a donor name. I would consider putting the two while loops into two different functions: email_menu and add_donor or something like that. Smaller functions with only one purpose are easier to test.